src/Main.js

```diff
@@ -145,19 +136,18 @@ class Main extends React.Component {
         </div>
         <div className="container-fluid row">
         <div className="col-sm-5">
-          <Testimony img="https://4.bp.blogspot.com/-h9fOaNMJsms/ToQ6XY_5__I/AAAAAAAAAf8/-DWR5VhIkC0/s1600/1178525479930832-cho-con-1-788229.jpg" name="John" grade="10th grade" content="This is my comment."/>
-          <Testimony img="https://4.bp.blogspot.com/-h9fOaNMJsms/ToQ6XY_5__I/AAAAAAAAAf8/-DWR5VhIkC0/s1600/1178525479930832-cho-con-1-788229.jpg" name="John" grade="10th grade" content="This is my comment."/>
+          <Testimony img="https://4.bp.blogspot.com/-h9fOaNMJsms/ToQ6XY_5__I/AAAAAAAAAf8/-DWR5VhIkC0/s1600/1178525479930832-cho-con-1-788229.jpg" name="Abigail" grade="9th grade" content="CollegePrep is easily the best place that I know for learning ACT and SAT. I would really like to thank all of you guys for making this possible!"/>
+          <Testimony img="https://4.bp.blogspot.com/-h9fOaNMJsms/ToQ6XY_5__I/AAAAAAAAAf8/-DWR5VhIkC0/s1600/1178525479930832-cho-con-1-788229.jpg" name="Bob" grade="10th grade" content="Thanks for creating this platform! I was able to get the help that I needed to improve my score, for free! The test would have been so much more difficult without this help."/>
```

Comment:
Let's create an assets folder on this server, put the image there and pass in a relative URL here!